addons/bus/static/tests/bus_test_helpers.js

```diff
@@ -26,6 +32,87 @@ export function defineBusModels() {
 
 export const busModels = { BusBus, IrWebSocket };
 
+export async function startServer() {
+    const { env } = await makeMockServer();
+    const pyEnv = env;
+    pyEnv["res.users"].write([serverState.userId], {
+        groups_id: pyEnv["res.groups"]
+            .search_read([["id", "=", serverState.groupId]])
+            .map(({ id }) => id),
+    });
```

Comment:
- I don't think the `pyEnv` constant is necessary.
- I'm not sure about the combination of `search_read` and `map`. It seems to only return an array containing the already known `serverState.groupId`. Am I missing something?

If that’s the case, I would prefer having this setup directly in the tests:

```js
const { env } = makeMockServer();
env["res.users"].write([serverState.userId], { groups_id: [serverState.groupId] });
```

IMO 2 lines of explicit setup are better than 10 of hidden setup.